src/functions/asyncCredential/asyncCredentialHandler.test.ts

```diff
@@ -446,13 +446,185 @@ describe("Async Credential", () => {
     });
   });
 
+  describe("Request body validation", () => {
+    describe("Given body is missing", () => {
+      it("Returns 400 status code with invalid_request_body error", async () => {
+        const event = buildRequest({
+          headers: { Authorization: `Bearer ${mockValidJwt}` },
+        });
+
+        dependencies.tokenService = () => new MockTokenSeviceValidSignature();
+
+        const result: APIGatewayProxyResult = await lambdaHandler(
+          event,
+          dependencies,
+        );
+
+        expect(result).toStrictEqual({
+          headers: { "Content-Type": "application/json" },
+          statusCode: 400,
+          body: JSON.stringify({
+            error: "invalid_request_body",
```

Comment:
Please may we just go with invalid_request unless the TD says otherwise

Reply:
Done